cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I guess it's mandatory or will be needed later on?

Reply:
It was part of the template that cypress installs automatically, haven't checked if it's really necessary but it'll likely be needed later, I'd keep it.